src/easystar.js

```diff
@@ -178,6 +180,19 @@ EasyStar.js = function() {
         iterationsPerCalculation = iterations;
     };
 
+    /**
+    * Sets the limit of the path queue instance index, which is decremented per
+    * calculation if enabled. If multiple calls to .findPath() are made, this
+    * will distribute the load by running iterationsPerCalculation before
+    * moving onto the next path in the queue.
+    *
+    * @param {Number} limit The maximum number of iterations before resetting
+    * on calculate() call. Use -1 to run all the paths at once.
+    **/
+    this.setParallelLimit = function(limit) {
```

Comment:
I'm trying to think of a use case where it makes sense to use anything other than `-1` or `1` as an argument to `setParallelLimit`. What do you think about making this something like `enableParallelCompute` and `disableParallelCompute`? This follows in line more with the `enableSync` option.

In v2 I'm thinking we should start accepting options like this in the constructor of `EasyStar` rather than these ad-hoc functions.

Reply:
if I submit 10 1 second paths and set the limit to -1, it will take ~10 seconds before it returns all the paths at once. if i set the limit to 2, it will return one path every 2 seconds instead. i also didn't need this feature, but I thought it could be useful to get paths returned quicker and still solve more than one, but maybe it's not worth being exposed here? I can replace it with the binary option for now.